dracut/modules.d/90kiwi-dump/kiwi-dump-image.sh

```diff
@@ -354,17 +354,17 @@ function get_remote_image_source_files {
 
     image_uri=$(getarg rd.kiwi.install.image=)
     image_md5_uri=$(
-        echo "${image_uri}" | awk '{ gsub("\\.xz",".md5", $1); print $1 }'
+      echo "${image_uri}" | awk '{ gsub("\\.xz",".md5", $1); gsub("dolly:","tftp:", $1); print $1 }'
```

Comment:
Can we keep the four space indentation model. I'm getting older and my eyes too :)

If you like to break the long line you can do that by e.g

```bash
    echo "${image_uri}" |\
        awk '{ gsub("\\.xz",".md5", $1); gsub("dolly:","tftp:", $1); print $1 }'
```

But maybe the following comment makes this one obsolete...